tests/include/test/psa_crypto_helpers.h

```diff
@@ -367,6 +367,30 @@ uint64_t mbedtls_test_parse_binary_string(data_t *bin_string);
 #define MD_PSA_DONE() ((void) 0)
 #endif /* MBEDTLS_MD_SOME_PSA */
 
+/** \def BLOCK_CIPHER_PSA_INIT
```

Comment:
\<rant>Aw, we're really starting to have too many of these macros. Nothing you can do about it now, I just can't wait for 4.0\</rant>